source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,181 @@
+.. _rm-timeseries:
+
+==================================
+Configure a Time Series Collection
+==================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure mapping rules to migrate a table that contains a 
+datetime column as a :ref:`time series collection <manual-timeseries-collection>`. 
```

Comment:
[question / suggestion]

I feel like "to" might make more sense than "as" here?

```suggestion
datetime column to a :ref:`time series collection <manual-timeseries-collection>`. 
```